packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -216,6 +226,23 @@ export class SceneTreeTableCell {
       !event.defaultPrevented &&
       event.button === 0 &&
       !this.interactionsDisabled
+    ) {
+      this.beginningPointerPosition = event.clientY;
+      window.addEventListener('pointerup', this.handleCellPointerUp);
```

Comment:
Instead of tracking the original position on `pointerdown` and having a separate `pointerup` handler, I'd suggest that we ditch the `pointerdown` altogether and just register a `pointerup` event. Note, that there's other logic in this component for calling `preventDefault` on some buttons, so those would need to be changed over to `pointerup`.

Also I think it's generally better if we relied on scroll events vs inferring scrolling through pointer positions. You could add something like this to the `scene-tree-table-layout` component to track if scrolling is currently happening, then setting an internal prop on the component to inform it that scrolling is happening and interactions should be disabled.

```ts
  private isScrolling = false;
  private scrollTimer: number | undefined;

  private handleScroll = (): void => {
    this.isScrolling = true;

    window.clearTimeout(this.scrollTimer);
    window.setTimeout(() => {
      this.isScrolling = false;
    }, 100);
  };
```

This could probably be simplified if we moved the selection/expansion/visibility calls to the layout component itself.

Lastly, I'd recommend that we not show hover states when on mobile. You can use `window.matchMedia("(pointer: coarse)").matches` to know if the user is using a touch device.